app/src/main/java/org/simple/clinic/phone/TwilioPhoneNumberMasker.kt

```diff
@@ -0,0 +1,40 @@
+package org.simple.clinic.phone
+
+import android.net.Uri
+import io.reactivex.Completable
+import io.reactivex.Single
+import org.simple.clinic.activity.TheActivity
+import javax.inject.Inject
+
+class TwilioPhoneNumberMasker @Inject constructor(
+    private val configProvider: Single<PhoneNumberMaskerConfig>,
+    private val activity: TheActivity
+) : PhoneNumberMasker {
+
+  override fun maskAndCall(numberToMask: String, caller: Caller): Completable {
+    return configProvider
+        .map { config -> maskNumber(config, numberToMask) }
+        .flatMapCompletable { maskedNumber -> callNumber(maskedNumber, caller) }
+  }
+
+  private fun maskNumber(config: PhoneNumberMaskerConfig, numberToMask: String): String {
+    return if (config.maskingEnabled) {
+      // TODO: This number is temporary. Every nurse will
+      // be assigned a number during login/registration.
+      val twilioNumber = "+1 704 452 4471"
```

Comment:
Replace with a fake number please.

Reply:
Whoops, keep forgetting that our source is open. Done.